package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.3.10",
   "versionCode": 21,
   "scripts": {
-    "start": "expo start --dev-client",
```

Comment:
can you explain why this is not needed?

Reply:
I'm not sure why this was here in the first place since the `--dev-client` flag should be the difference between `start` and `start:native` [according to the README](https://github.com/nearform/optic-expo#notes)